fetcher/fetcher.go

```diff
@@ -111,7 +112,13 @@ func (bot *fetcherImpl) catchupLoop(ctx context.Context) error {
 	var blockbytes []byte
 	aclient := bot.Algod()
 	for {
+		start := time.Now()
+
 		blockbytes, err = aclient.BlockRaw(bot.nextRound).Do(ctx)
+
+		dt := time.Since(start)
+		metrics.AlgodRawBlockTimeSeconds.Observe(dt.Seconds())
```

Comment:
Could this metric be named `GetAlgodRawBlockTimeSeconds` instead? That might be easier to understand when building queries in grafana